histolab/util.py

```diff
@@ -345,13 +357,13 @@ def lazyproperty(f: Callable[..., Any]):
     The parameter names in the methods below correspond to this usage
     example::
 
-        class Obj(object):
+                    class Obj(object):
 
-            @lazyproperty
-            def fget(self):
-                return 'some result'
+                                    @lazyproperty
+                                    def fget(self):
+                                                    return 'some result'
 
-        obj = Obj()
+                    obj = Obj()
```

Comment:
each new commit is adding further spaces. Could you please check what's the problem?

Reply:
It could be 😄 thanks ❤️